### PR TITLE
Wrap the deferred timer in a function

### LINKED ISSUE #341: Wrap the deferred timer in a function
The statement after the `defer` is evaluated at the time of the first encounter. For example,

https://github.com/sql-machine-learning/sqlflow/blob/ff96c186817ef5aefb1f1cbe0e0dce9176f379ad/sql/executor.go#L177-L179

This code snippet will always log `elapsed: 0 second`. The correct usage is to wrap the deferred timer in a function, such as
```go
defer func() {
        fmt.Printf("elapsed:%v", time.Since(startAt))
}()
```

### PR DESCRIPTION
**Pull request by tonyyang-svail**
Fix #341

### CODE REVIEW

File: sql/executor.go [hunk 1]
```diff
@@ -134,7 +136,10 @@ func runExec(slct string, db *DB) *PipeReader {
 
 		err := func() error {
 			startAt := time.Now()
-			log.Infof("Starting runStanrardSQL1:%s", slct)
+			log.Infof("Starting runExec:%s", slct)
```
Comment (typhoonzero) [inline sql/executor.go]: Maybe we only need time elapsed for the `slct` statement in the log, so this line can be `Debugf`? And below elapsed log can log together with `slct`, or it maybe harder to figure out which statement this time was.

Comment (tonyyang-svail) [reply sql/executor.go]: Sounds good. :)

File: sql/executor.go [hunk 2]
```diff
@@ -55,8 +55,9 @@ func runQuery(slct string, db *DB) *PipeReader {
 		defer wr.Close()
 
 		err := func() error {
-			startAt := time.Now()
-			log.Infof("Starting runStanrardSQL:%s", slct)
+			defer func(startAt time.Time) {
+				log.Debugf("runQuery finished, elapsed:%v", time.Since(startAt))
```
Comment (typhoonzero) [inline sql/executor.go]: Maybe also output the `slct` string?

**Review by typhoonzero — APPROVED** [review]

File: sql/executor.go (hunk 2 above)
Comment (tonyyang-svail) [reply sql/executor.go]: My bad..

**Pull request merged**